tests/page-objects/reports/reports.wdio.page.js

```diff
@@ -42,6 +42,9 @@ const checkCss = 'input[type="checkbox"]';
 const sentTask = async () => (await reportBodyDetails()).$('ul .task-list .task-state .state');
 const reportByUUID = (uuid) => $(`li[data-record-id="${uuid}"]`);
 
+const patientName = () => $('.subject .name');
+const reportType = () => $('#reports-content > div.body.report-body > div.item-summary > div > div:nth-child(2)');
```

Comment:
Similar to my note above about fragile css selectors, this one is kind of fragile. Also, for the parts like `.report-body` and `.item-summary' here, you don't need to require that the class is on any particular element (that .div prefix).